qiskit/ignis/experiments/calibration/instruction_data/database.py

```diff
@@ -98,7 +110,9 @@ def get_generator_kwargs(
         Args:
             qubits: Index of qubit(s) to search for.
             channel: Label of pulse channel to search for.
-            gate_type: Name of gate to search for.
+            inst_name: Name of gate to search for.
+            pulse_type: Name of ParametricPulse generator that is used for pulse creation.
```

Comment:
This is the name of the pulse envelope right? If so I don't think we should use the `generator` has it already has a strong meaning in the cal module.

Reply:
Can you fix this in the other comments too?